backend/utils/constants.ts

```diff
@@ -0,0 +1,4 @@
+// Purpose: Contains all the constants used in the application.
```

Comment:
We could probably merge the 2 constants.ts files, there is another one containing default org role permissions in `backend/constants.ts`